src/ua/hillel/FindWordPosition.java

```diff
@@ -0,0 +1,17 @@
+package ua.hillel;
+
+public class FindWordPosition {
+    public static void main(String[] args) {
+
+        findWordPosition();
+    }
+
+    public static void findWordPosition() {
+
+        String source = "method";
+        String target = "et";
+        int position = source.indexOf(target);
+
+        System.out.println("Перша строка: " + source + ". " + "Друга строка: " + target + ". " + "Позиція: " + position);
```

Comment:
We do not use the `print()` statement in methods, only in `main` for testing purposes. Please remove it from here.